gating/mnaio_vars.sh

```diff
@@ -16,9 +16,9 @@ fi
 #
 # Environment settings
 #
-if [[ "${RE_JOB_IMAGE}" == "xenial" ]]; then
+if [[ "${RE_JOB_IMAGE}" =~ .*xenial.* ]]; then
```

Comment:
Should we drop the quotes here - if not for function, for consistency?

Reply:
Yeah, probably.

Reply:
ID-10-T Error, I see what you're saying, yes I think we should.  I'll update the PR